arktype.io/src/featuresConfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I do like abstracting this out to an object, but given the limited size and the fact that it is unlikely to grow much more, I'd likely include it in TS alongside the component that consumes it.